makefiles/release.mk

```diff
@@ -213,10 +213,18 @@ bump-fluvio: install-fluvio-package
 bump-fluvio-stable: CHANNEL_TAG=stable
 bump-fluvio-stable: VERSION=$(REPO_VERSION)
 bump-fluvio-stable: bump-fluvio
+	export PKGSET_NAME=$(VERSION)
+	export FLUVIO_VERSION=$(VERSION)
+	export FLUVIO_CLOUD_VERSION="0.2.15"
```

Comment:
If we think we'll realistically be maintaining static metadata values in the makefiles, we should move it up to the top of the file

Reply:
Totally! This is temporal approach while we solve in the service side.